public/utils/helpers.tsx

```diff
@@ -203,7 +203,6 @@ export function renderVisualization(spec: TopLevelSpec, containerId: string) {
           : 'transparent';
 
         const firstItem = values.pop() || ['', ''];
-
```

Comment:
Was this a manual edit or the linter removed this line? If manual, we can avoid it since it only pollutes the file history